<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>737</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_temp">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Temperatura BMP</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_pres">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Presion</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_hum">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Humedad</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_estado">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Estado</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_temp">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_pres">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_hum">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>160</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_estado">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>200</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_T">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>RecibirTemp_bmp</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_P">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>110</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>RecibirP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_conectar">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>200</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Conectar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Leer_dht">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Leer DHT</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_puertos">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>200</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="button_refresh_ports">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Refrescar puertos</string>
    </property>
    <property name="statusTip">
     <string>Refrescar puertos</string>
    </property>
    <property name="text">
     <string>Actualizar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_P_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>160</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>RecibirH</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_leerdht">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>671</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_temp_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Temperatura DHT</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_temp_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_T_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>60</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>RecibirTemp_dht</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Leer_bmp">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>211</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Leer BMP</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_leerbmp">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>671</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
